packages/sanity/src/structure/panes/document/DocumentPaneProvider.tsx

```diff
@@ -749,11 +749,13 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
 
   useEffect(() => {
     if (connectionState === 'reconnecting') {
-      pushToast({
-        id: 'sanity/structure/reconnecting',
-        status: 'warning',
-        title: t('panes.document-pane-provider.reconnecting.title'),
-      })
+      setTimeout(() => {
```

Comment:
This needs to be cancelled in an effect cleanup

Reply:
Oops, sorry I forgot to add it, should be there now!